pages/ActivityStreams/Overview.md

```diff
@@ -33,22 +33,347 @@ These deviations from the standard will be thoroughly documented here.
+  },
+  "links": [
+    {
+      "type": "Link",
+      "name": "John Doe's personal site",
+      "href": "https://example.org"
+    },
+    {
+      "type": "Link",
+      "name": "John Doe's GitHub account",
+      "href": "https://github.com/example"
+    }
+  ]
+}
+```
+
+#### Video
+
+Clients are recommended to support presentation for activity objects of type video to represent video content posted by users.
+Video type objects, if supported, must at minimum contain a `"type"` field with the value `"Video"` and a `"URL"` with either a URL string or an activity sub-object of type link containing a URL pointing to a video file.
+
+Video objects are also recommended to include a `"mediaType"` field on their link sub-object with a standard [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) string value.
+If no media type is provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+
+Clients that support video objects must at least support media types of `"video/mpeg"` complying with [RFC2045](https://www.iana.org/go/rfc2045), `"video/ogg"` complying with [RFC5334](https://www.iana.org/go/rfc5334) and `"video/webm"` complying with the [WebM standard](https://www.webmproject.org/docs/container/).
+Clients are also recommended to support `"video/H256"` complying with [RFC7798](https://www.iana.org/go/rfc7798), `"video/mp4"` complying with [RFC4337](https://www.iana.org/go/rfc4337) and `"video/raw"` complying with [RFC4175](https://www.iana.org/go/rfc4175).
+
+Video objects may also include a `"duration"` field with a string value complying with the [XML Schema 11-2](https://www.w3.org/TR/xmlschema11-2/) standard for duration strings as recommended in the [Activity Vocabulary](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-duration) specification.
+Video objects linking to live streaming content may use the string `"Live"` for the duration field or omit it entirely.
+
+Video objects may include a `"preview"` field with an activity sub-object of type link, a URL string or an array of sub-objects and/or strings referring to shorter video files or image files provided as a preview of longer video files.
+It is recommended that implementers at minimum include one image previews wherever possible.
+Preview images should generally include at least one file of the following resolutions: `320x240`, `640x480` or `1024x768`.
+
+Clients that support video objects should also include fields of `"height"` and `"width"` each with positive integer value on either the root activity object or the individual link sub-objects if multiple are present.
+If these fields are not provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+Clients may also ignore video files of exceptionally large or small sizes, however videos with a preview of one of the prior mentioned resolutions must always be supported.
```

Comment:
The preview field might also be exceptionally large or small. I'm not sure we want to make this a requirement.